packages/web/src/store/errors/sagas.ts

```diff
@@ -14,6 +17,11 @@ function* handleError(action: errorActions.HandleErrorAction) {
       error: new Error(action.message),
       name: action.name
     })
+    yield put(
+      make(Name.APP_ERROR, {
+        errorMessage: action?.message ?? 'Unknown Error'
```

Comment:
Cool, how often does this happen relative to other stuff btw?

Reply:
Not sure I follow, relative to what do you mean? Like different errors or what?